bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,274 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-new.html" %}
+
+{% add_lang_files "mozorg/home/index" %}
+
+{% block gtm_page_id %}data-gtm-page-id="Homepage"{% endblock %}
+
+{% block page_image %}{{ static('img/home/page-image.png') }}{% endblock %}
+
+{% block page_title %}
+  {{ _('Internet for people, not profit') }}
+{% endblock %}
+
+{% block page_desc %}
+  {{ _('Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource, open and accessible to all.') }}
+{% endblock %}
+
+{% block body_id %}home-b{% endblock %}
+{% block body_class %}home-variant home-variant-b{% endblock %}
+
+{% block extra_meta %}
+<!-- validates bing webmaster tools -->
+<meta name="msvalidate.01" content="B7B177115A634927D608514DA17B2574">
+<!-- YouTube Verification -->
+<meta name="google-site-verification" content="U9a6gH32vLIykvntaDToj-ytYhlZ1AfAgVEKstixQIE">
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'home-b' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('mozorg-home-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}
+<header class="section masthead" id="masthead">
+  <div class="content">
+
+  {% block tabzilla_tab %}{% endblock %}
```

Comment:
Is this empty block needed seeing as you're overriding `{% block site_header %}`?